helm_deploy/values-dev.yaml

```
@@ -11,6 +11,7 @@ generic-service:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: "applicationinsights.dev.json"
     HMPPS_AUTH_URL: "https://sign-in-dev.hmpps.service.justice.gov.uk/auth"
     FRONTEND_URL: "https://breach-notice-dev.hmpps.service.justice.gov.uk"
+    NDELIUS-INTEGRATION-API_URL: "breach-notice-and-delius-dev.hmpps.service.justice.gov.uk"
```

Sorry it got missed off my message but you'll need to include the protocol for these:
```suggestion
    NDELIUS-INTEGRATION-API_URL: "https://breach-notice-and-delius-dev.hmpps.service.justice.gov.uk"
```

---

Updated.